package.json

```diff
@@ -1,23 +1,23 @@
 {
-  "name": "express-typescript-boilerplate",
-  "version": "1.0.14",
-  "description": "An Express boilerplate backend",
   "author": "Edwin Hernandez",
-  "repository": "edwinhern/express-typescript-2024",
+  "description": "An Express boilerplate backend",
   "license": "MIT",
-  "main": "index.ts",
+  "name": "express-typescript-boilerplate",
   "private": true,
+  "repository": "edwinhern/express-typescript-2024",
   "scripts": {
-    "dev": "tsx watch --clear-screen=false src/index.ts | pino-pretty",
     "build": "tsup",
-    "start": "node dist/index.js",
     "clean": "rimraf dist coverage",
+    "dev": "tsx watch --clear-screen=false src/index.ts | pino-pretty",
+    "format": "biome format src/",
     "lint": "biome check src/",
     "lint:fix": "biome check src/ --fix",
-    "format": "biome format src/",
-    "test": "vitest run",
-    "prepare": "husky"
+    "prepare": "husky",
+    "setup": "cp .env.example .env && npm install && npm run dev",
```

Comment:
This is an easy 1 liner that _should_ copy the .env for you, get the dependencies, and start your dev server.